public/javascripts/songlistTable.js

```diff
@@ -0,0 +1,66 @@
+var addOrDeleteSong = function(id) {
```

Comment:
This method showed up earlier too. Consider moving it to it's own javascript library if it is re-used and just include it on the page that includes this. (there are better ways to handle it, but we don't need to worry about that yet for now).